Add target problem and persona to main Company page (#293)
meltano/handbook: src/_company/index.md

## Vision

> Our vision is for Meltano to become **the foundation of every team's ideal data platform**.

Author: ```suggestion
Our vision is for Meltano to be **the foundation of every team's ideal data platform**.
```

More concretely, we are building **open source DataOps Platform Infrastructure** that will bring the benefits of software development best practices such as DevOps and open source to the entire data lifecycle.

To learn more about this vision, check out this [detailed blog post](https://meltano.com/blog/meltano-the-strategic-foundation-of-the-ideal-data-stack/#vision) by our CEO Douwe Maan.

## Persona and Problem

Our mission and vision are broad. To be able to achieve them, we must find product-market fit with a targeted profile of users so that we can build a sustainable business and continue to grow. 

Our target persona are mid- to senior-level Data Engineers who work for companies with 30-200 people and are likely the only data professional in their company.

> The problem we are solving for them is that they are in the process of setting up a new data platform that will enable their organization to generate valuable insights, but they are dissatisfied with their previous experience and want a solution that addresses many of the problems they encountered as their previous platform attempted to change and grow. Their previous experience likely include:

Author: ```suggestion
The problem we are solving for them is that they are in the process of selecting different tools, what we would call "setting up a new data platform", that will enable their organization to generate valuable insights, but they are dissatisfied with their previous experience and want a solution that addresses many of the problems they encountered as their previous platform attempted to change and grow. Their previous experience likely include:
```

@emilylucie what do you think about a caveat / change like this?

Reviewer: I like "data stack" better than "data platform", as I think data engineers do think of "picking tools" as equivalent to "putting together a data stack"

```suggestion
The problem we are solving for them is that they are in the process of putting together a new data stack that will enable their organization to generate valuable insights, but they are dissatisfied with their previous experience and want a solution that addresses many of the problems they encountered as their previous platform attempted to change and grow. Their previous experience likely include:
```
